diesel/src/pg/query_builder/distinct_on.rs

```diff
@@ -39,6 +39,7 @@ where
             self.limit_offset,
             self.group_by,
             self.locking,
+            self.having,
```

Comment:
Shouldn't this impl be also generic over the having clause?

Reply:
Indeed it should, thanks! Fixed